SwiftSyntaxGeneration/Package.swift

```diff
@@ -0,0 +1,41 @@
+// swift-tools-version:5.6
+
+import PackageDescription
+
+let package = Package(
+  name: "SwiftSyntaxGeneration",
+  products: [
+    .executable(name: "swift-syntax-builder-generation", targets: ["swift-syntax-builder-generation"]),
```

Comment:
Minor note: The reason for renaming `SwiftSyntaxBuilderGeneration` to `swift-syntax-builder-generation` (kebab-case) is twofold:

1. Until the pinned revision of `swift-syntax` no longer has a target named `SwiftSyntaxBuilderGeneration` the name will conflict with this target
2. Most Swift repos follow the convention of naming executable targets in kebab-case (e.g. this repo's `lit-test-helper`, SourceKit-LSP's `sourcekit-lsp`, SwiftFormat's `swift-format`, ...). Therefore this would be good opportunity to update the name.